src/test/java/com/github/alexcojocaru/mojo/elasticsearch/LoadElasticsearchDataMojoTest.java

```diff
@@ -41,6 +47,7 @@ protected void setUp() throws Exception
         
         // I cannot find another way of setting the two required propperties at run time.
         mojo.httpPort = esPorts.get(ElasticsearchPort.HTTP);
+
```

Comment:
Can you also add a test to this file, to verify that the skip flag skips the loading?